recipe_scrapers/grandfrais.py

```diff
@@ -0,0 +1,100 @@
+# mypy: allow-untyped-defs
+
+
+import re
+from urllib.parse import urlparse
+
+from ._abstract import AbstractScraper
+from ._utils import FRACTIONS, get_minutes, normalize_string
+
+
+class GrandFrais(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "grandfrais.com"
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return None
+
+    def total_time(self):
+        cook_time, prep_time = self.cook_time(), self.prep_time()
+        if cook_time and prep_time:
```

Comment:
It looks like the site includes some recipes that don't require any cooking time -- cold salads, and similar dishes -- so I think we might want a second condition:

```
elif prep_time:
    return prep_time
```

Reply:
Wow, good catch. I hadn't seen those.
I'll chech a bit deeper if we can have a `cook_time` without `prep_time` _(weird, but perhaps possible with ready-made dishes)_